### PR TITLE
Supporting .NET8 updates

### PR DESCRIPTION
**Pull request by paulyuk**
# Description

These complementary updates do the following:
- fix Actors tests by changing test match string of healthz endpoint to latest value
- fix Actors tests with correct paths to do `dapr run` commands (no more path not found)
- update Bindings to .NET8 and clean out swashbuckle references

## Issue reference

[#980](https://github.com/dapr/quickstarts/issues/980)
[#987](https://github.com/dapr/quickstarts/issues/987)

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] The quickstart code compiles correctly
* [x] You've tested new builds of the quickstart if you changed quickstart code
* [x] You've updated the quickstart's README if necessary
* [x] If you have changed the steps for a quickstart be sure that you have updated the automated validation accordingly. All of our quickstarts have annotations that allow them to be executed automatically as code. For more information see [mechanical-markdown](https://github.com/dapr/mechanical-markdown#readme). For user guide with examples see [Examples](https://github.com/dapr/mechanical-markdown/tree/main/examples#mechanical-markdown-by-example).

### CODE REVIEW

File: actors/csharp/sdk/README.md [hunk 1]
```diff
@@ -34,15 +34,15 @@ Run the `SmartDevice.Service`, which will start service itself and the Dapr side
 <!-- STEP
 name: Run actor service
 expected_stdout_lines:
-  - "Request finished HTTP/1.1 GET http://127.0.0.1:5001/healthz - - - 200"
+  - "Request finished HTTP/1.1 GET http://127.0.0.1:5001/healthz - 200"
```
Comment (paulyuk) [inline actors/csharp/sdk/README.md:37]: This is the line of output from healthz that changed, causing our tests to fail.  by changing the expected out we fix it back up again.

File: .github/env/global.env [hunk 2]
```diff
@@ -1,5 +1,5 @@
       DAPR_CLI_VERSION: 1.12.0
-      DAPR_RUNTIME_VERSION: 1.13.0-rc.2
+      DAPR_RUNTIME_VERSION: 1.13.0-rc.4
```
Comment (paulyuk) [inline .github/env/global.env:2]: hopefully this avoids a merge conflict with release-1.13.

File: bindings/csharp/http/batch/batch.csproj [hunk 3]
```diff
@@ -2,13 +2,9 @@
...
     <PropertyGroup>
       <OutputType>Exe</OutputType>
-      <TargetFramework>net6.0</TargetFramework>
+      <TargetFramework>net8.0</TargetFramework>
       <Nullable>enable</Nullable>
       <ImplicitUsings>enable</ImplicitUsings>
     </PropertyGroup>
-  
-    <ItemGroup>
-      <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
```
Comment (paulyuk) [inline bindings/csharp/http/batch/batch.csproj:11]: Reviewed with .NET SDK maintainers in Dapr and they agree with this.  Also tests pass.

File: bindings/csharp/sdk/batch/batch.csproj [hunk 4]
```diff
@@ -2,13 +2,12 @@
...
     <PropertyGroup>
       <OutputType>Exe</OutputType>
-      <TargetFramework>net6.0</TargetFramework>
+      <TargetFramework>net8.0</TargetFramework>
       <Nullable>enable</Nullable>
       <ImplicitUsings>enable</ImplicitUsings>
     </PropertyGroup>
   
     <ItemGroup>
-      <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
```
Comment (paulyuk) [inline bindings/csharp/sdk/batch/batch.csproj:11]: Reviewed with .NET SDK maintainers in Dapr and they agree with this.  Also tests pass.

**Review by pngan — APPROVED** [review]
LGTM

**Pull request merged**